### ISSUE TITLE
Forbid multi-line for and while loops

### ISSUE THREAD

**Issue by sobolevn** [wemake-services/wemake-python-styleguide#813]
# Rule request

## Thesis

Code [like this](
https://github.com/sobolevn/itmo-2019/pull/24/files#diff-5ad37df7dcdec8d4aeb51aac0bde8387R51-R54) should be forbidden: 

```python
    for test_file in find_files(
        target_dir,
        '{0}*{1}'.format(TEST_PREFIX, PY_EXTENSION),
    ):
        module = import_module(test_file)
```

## Reasoning

`for`s should be simple and readable. The correct way to rewrite the same code is:

```python
    files = find_files(
        target_dir,
        '{0}*{1}'.format(TEST_PREFIX, PY_EXTENSION),
    )
    for test_file in files:
        module = import_module(test_file)
```

This is a consistency rule for `ast.For`, `ast.AsyncFor`, and `ast.While` nodes.

**Comment by eternalgum** [comment]
I think I can handle it

**Comment by sobolevn** [comment]
Assigned! Thanks a lot, @eternalgum!

**Comment by sobolevn** [comment]
@abyss143 feel free to ping me in case you are still on it.

**Comment by Ben-Drebing** [comment]
Happy to try this out :)

**Comment by sobolevn** [comment]
Thanks 👍

**Comment by Ben-Drebing** [comment]
@sobolevn Quick question. Should I be adding the check for this violation to the `WrongLoopVisitor`? It looks like that visitor will get the job done, but I wanted to make sure that is the correct one to use.

**Comment by sobolevn** [comment]
Yes, looks fine
